README.rst

```diff
@@ -424,6 +428,82 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
      - ``lowest``
      - Decides on what basis the Karapace cluster master is chosen (only relevant in a multi node setup)
 
+Karapace Schema Registry authorization file is an optional JSON configuration, which contains a list of authorized users in ``users`` list and a set of access control rules in ``permissions`` list.  Karapace uses HTTP Basic Authentication for user authentication.
+
+Each user entry contains following attributes:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``username``
+     - A string
+   * - ``algorithm``
+     - One of supported hashing algorithms, ``scrypt``, ``sha1``, ``sha256``, or ``sha512``
+   * - ``salt``
+     - Salt used for hashing the password
+   * - ``password_hash``
+     - Hash string of the password calculated using given algorithm and salt.
+
+Password hashing can be done using ``karapace_mkpasswd`` tool, if installed, or by invoking directly with ``python -m karapace.auth``. The tool generates JSON entry with these fields. ::
+
+  $ karapace_mkpasswd -u user -a sha512 secret
+  {
+      "username": "user",
+      "algorithm": "sha512",
+      "salt": "iuLouaExTeg9ypqTxqP-dw",
+      "password_hash": "R6ghYSXdLGsq6hkQcg8wT4xkD4QToxBhlp7NerTnyB077M+mD2qiN7ZxXCDb4aE+5lExu5P11UpMPYAcVYxSQA=="
+  }
+
+Each access control rule contains following attributes:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``username``
+     - A string to match against authenticated user
+   * - ``operation``
+     - Exact value of ``Read`` or ``Write``. Write implies also read permissions. Write includes all mutable operations, e.g. deleting schema versions
+   * - ``resource``
+     - A regular expression used to match against accessed resource of form ``Subject:<subject_name>`` or ``Config:<optional_subject_name>``
+
+Example of complete authorization file
```

Comment:
As said earlier, configuring the auth file should have a heading and this would then be a sub-header.